tests/plugins/test_integralgrader.py

```diff
@@ -516,7 +517,7 @@ def test_debug_message():
         "==============================================<br/>\n"
         "Integration Data for Sample Number 0<br/>\n"
         "==============================================<br/>\n"
-        "Variables: {{'infty': inf, 'e': 2.718281828459045, 'i': 1j, 'j': 1j, 's': 5.530375019455111, 'x': 5.530375019455111, 'pi': 3.141592653589793}}<br/>\n"
+        "Variables: {{{u}'infty': inf, {u}'e': 2.718281828459045, {u}'i': 1j, {u}'j': 1j, u's': 5.530375019455111, u'x': 5.530375019455111, {u}'pi': 3.141592653589793}}<br/>\n"
```

Comment:
Wow, also unpleasant!

Is there a reason that s and x get `u` instead of `{u}`?

Reply:
fixed